tasks/get_current_archive_list/test/unit_tests/test_get_current_archive_list.py

```diff
@@ -957,7 +964,7 @@ def test_handler_happy_path(
         """
         Happy path for handler assembling information to call Task.
         """
-        secret_arn = uuid.uuid4().__str__()
+        db_connect_info_secret_arn = uuid.uuid4().__str__()
```

Comment:
Looks like find/replace error. The get_s3_credentials calls should not use something called db_connect_info_secret_arn as it could lead to confusion. I would either revert to the generic `secret_arn` or something more specific like `s3_credentials_secret_arn`. This applies to lines 967, 1004, 1011, 1047, 1077

Reply:
yes changed to mock_s3_credentials_secret_arn